yt/docker/targets.hcl

```diff
@@ -0,0 +1,71 @@
+variable "REPO_PATH" {}
+
+variable "BUILD_ROOT" {
+  default = "${REPO_PATH}/yt/docker"
+}
+
+
+# Common targets
+
+target "built_yson" {
+  args = {
+    BUILD_TARGETS = "yson_lib driver_lib driver_rpc_lib"
+  }
+  contexts = {
+    base_img = "docker-image://ubuntu:jammy"
+  }
+
+  dockerfile = "${BUILD_ROOT}/bake/built_yt.Dockerfile"
+  context = "${REPO_PATH}"
+}
+
+target "built_yt_all" {
```

Comment:
Let name it `yt_all`?

I don't understand why we want to precede the name with prefix `built_`